orca-api-tck/src/main/kotlin/com/netflix/spinnaker/orca/api/test/OrcaFixture.kt

```diff
@@ -29,7 +35,22 @@ import org.springframework.test.context.TestPropertySource
 @SpringBootTest(classes = [Main::class])
 @ContextConfiguration(classes = [TestKitConfiguration::class])
 @TestPropertySource(properties = ["spring.config.location=classpath:orca-test-app.yml"])
-abstract class OrcaFixture
+abstract class OrcaFixture {
```

Comment:
In the downstream plugin repo I am doing:

```
@TestPropertySource(properties = ["spinnaker.extensibility.plugins.netflix.winston.enabled=true"])
class Fixture : OrcaFixture()
```

This works, but I'd like to provide a better experience so plugin developers don't need to think about the config structure (`spinnaker.extensibility.plugins`) and can just provide the ID and we wire things up for them.